### PR TITLE
Fixing php issues reported by PHP EA Inspection

### PR DESCRIPTION
**Pull request by pascalseeland**
@smeyer-ilias

I did remove some code from the ilSoapUtils which I think is a leftover and was partially already removed. Could you please check an if this fine with you and if yes please merge this pr.

### CODE REVIEW

**Comment by mjansenDatabay** [comment]
@smeyer-ilias It would be great if you could review/merge the changes from @pascalseeland , because my review of the `ECS` component in the `PHP 8 Project` depends on it. I would like to prevent redundant reviews.

And: Maybe #4276 should be merged/handled first, because this PR here modifies a SOAP file (https://github.com/ILIAS-eLearning/ILIAS/pull/4282/files#diff-ebdb670209e355cc813aeccb415126c4049ddb93f69d972502f808d66e105ba0).

File: webservice/soap/classes/class.ilSoapUtils.php [hunk 1]
```diff
@@ -537,31 +537,6 @@ private function linkNode(array $node, ilCopyWizardOptions $cp_options) : int
         return $new_ref_id;
     }
 
-    /**
-     * @todo get rid of the soap wrapper
-     */
-    public function handleECSTasks(string $sid, int $a_server_id)
```
Comment (smeyer-ilias) [inline webservice/soap/classes/class.ilSoapUtils.php:537]: It's not just this method that needs to be removed. In addition, the WSDL generation and other places must also be adjusted. I will create a PR for this.

**Review by smeyer-ilias — APPROVED** [review]
The changes in the SOAP subclass are fine for me.

**Comment by smeyer-ilias** [comment]
There was only one remaining part in class.ilNusoapAdministrationAdapter.php
I fixed this in trunk

**Pull request merged**